garbage-collect/handler.go

```diff
@@ -46,20 +58,19 @@ func Handle(req []byte) string {
 	}
 
 	fmt.Printf("Functions owned by %s:\n %s", owner, deployedFunctions)
-
+	client := faasSDK.NewClient(&FaaSAuth{}, gatewayURL, nil, &timeout)
 	deleted := 0
 	for _, fn := range deployedFunctions {
 		if garbageReq.Repo == "*" ||
 			(fn.GetRepo() == garbageReq.Repo && !included(&fn, owner, garbageReq.Functions)) {
-
-			err = deleteFunction(fn.Name, gatewayURL)
+			fmt.Println("Delete ", fn.Name)
+			err = client.DeleteFunction(context.Background(), fn.Name, "")
```

Comment:
What is the empty string? If it’s a namespace, declare a local var called that with the empty string or openfaas-fn.

Reply:
yes we can do that. should not we use this value as "" and let API figure out the default namespace rather than we passing that value ?

Reply:
At least name the env? Passing “” means nothing when reading

Reply:
yeah sure I will move that to a variable